package.json

```diff
@@ -23,7 +23,7 @@
     "typescript": "^5.0.4"
   },
   "dependencies": {
-    "@blend-capital/blend-sdk": "^0.4.0",
+    "@blend-capital/blend-sdk": "file:../blend-sdk-js",
```

Comment:
- [ ] we should deploy the necessary version of `blend-sdk` and update this PR before